src/components/SatelliteManifestPanel/SatelliteManifestPanel.tsx

```diff
@@ -206,58 +246,83 @@ const SatelliteManifestPanel: FunctionComponent<SatelliteManifestPanelProps> = (
     setRowExpandedStatus(newRowExpandedStatus);
   };
 
+  const openCurrentEntitlementsListFromPanel = () => {
+    closeDetailsPanel();
+
+    const newRowExpandedStatus = [...rowExpandedStatus];
+    newRowExpandedStatus[currentDetailRowIndex] = true;
+    setRowExpandedStatus(newRowExpandedStatus);
+  };
+
   const collapseAllRows = () => {
     const newRowExpandedStatus = new Array(10).fill(false);
     setRowExpandedStatus(newRowExpandedStatus);
   };
 
+  // The ternary here is to avoid calling the API when collapsed.
+  const panelContent = detailsDrawerIsExpanded ? (
```

Comment:
I actually ended up having to cut this; the lack of animation with the side drawer was caused by having no content at the beginning (the blank part of the ternary). I just left it as an empty react fragment so that the type was React.ReactNode either way (what Patternfly's docs say should be passed into panel content). But the point is moot now.

Reply:
That makes sense.  I didn't think about maintaining the type of the response.

Hooray for opportunities to say "moot"!